lib/Mojolicious/resources/public/mojo/highlight.js/highlight-mojo-light.css

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Why does this file exist?

Reply:
Yep, this shouldn't be here, removed just now.